src/Azure/Functions/GithubIssues/GithubIssues/__init__.py

```diff
@@ -24,7 +25,7 @@ def __init__(self, user):
     # method to fetch the Auth0 management API token	
     def get_token(self):	
         # payload and header	
-        auth0_payload = "{\"client_id\":\"RKkiHSDtTGQWijgy7q7Pi7sheonzWZke\",\"client_secret\":\"pzoI_Zwp4i-PSlb0lHVOZTyl8zZ8iQmuzJ8GNfR4pY1Ttr1PgXPK3Bbk-LdRXyf9\",\"audience\":\"https://psa-tech-dashboard.auth0.com/api/v2/\",\"grant_type\":\"client_credentials\"}"	
```

Comment:
Is it possible to add these `secrets` outside Github?

Reply:
that is a good point. it should be possible

Reply:
moved to env variable